homeassistant/components/transmission/const.py

```diff
@@ -23,14 +23,17 @@
     ),
 }
 CONF_ENTRY_ID = "entry_id"
+CONF_ENTRY_VERSION = 2
```

Comment:
Please specify the version directly in config_flow.py

Reply:
Changed it and its now in the config_flow